fedimint-api/src/db/notifications.rs

```diff
@@ -0,0 +1,230 @@
+use std::{
+    collections::hash_map::DefaultHasher,
+    hash::{Hash, Hasher},
+};
+
+use async_trait::async_trait;
+use bitvec::vec::BitVec;
+use tokio::sync::{futures::Notified, Notify};
+
+use super::Result;
+use super::{IDatabaseTransaction, PrefixIter};
+
+/// Number of buckets used for `Notifications`.
+const NOTIFY_BUCKETS: usize = 32;
+
+/// Manages notifications for the database
```

Comment:
"Manages" is vague, and it's unclear what do we mean by "notifications". Is this on the notification sending site, or notification receiving one?